Add profile

ngencokamin opened the pull request:
Added my profile. I used a split image to show both the menu art and the splash, added a description bc it feels not self-explanatory, and added a "textures" box bc it has the game cartridge/disk too

gabrielfvale reviewed (changes requested):
I love the template and the preview!
Adding a description is a good idea, I might just snag that and add it to the submission guide.
One minor gripe I have though: I get why you added a "textures" badge, but I mostly intended on leaving those to specify which outputs a template supports. Maybe smaller tags could work?

ngencokamin commented:
> I love the template and the preview! Adding a description is a good idea, I might just snag that and add it to the submission guide. One minor gripe I have though: I get why you added a "textures" badge, but I mostly intended on leaving those to specify which outputs a template supports. Maybe smaller tags could work?

Do you have a preference between me shrinking the tags vs just removing the textures tag? Happy to do either

ngencokamin commented:
> I love the template and the preview! Adding a description is a good idea, I might just snag that and add it to the submission guide. One minor gripe I have though: I get why you added a "textures" badge, but I mostly intended on leaving those to specify which outputs a template supports. Maybe smaller tags could work?

Okay so I removed the box and added an "Assets Used" section with all the elements the profile uses. Is that better? I also drafted a new template that I can add to the PR if you're interested

```md
|-------------|-------------|
| <img src="https://placehold.co/640x480" width="320"> | **Template**: [Enter Name] <br> **Description**: [Enter a brief description of the profile] <br> **Author**: [Enter Author Name] <br> **Date of Submission**: [Enter Date] <br> **Download**: [your_template.zip](files/your_template.zip) <br> **Assets Used**: [Skyscraper Elements Used] (optional) <br><br> ![BOX](https://img.shields.io/static/v1?label=&message=BOX&color=%23EDD113&style=for-the-badge) ![PREVIEW](https://img.shields.io/static/v1?label=&message=PREVIEW&color=%23EDD113&style=for-the-badge) ![SPLASH](https://img.shields.io/static/v1?label=&message=SPLASH&color=%23EDD113&style=for-the-badge) <br> |
```
![snapshot_2025-03-21_20-24-11](https://github.com/user-attachments/assets/68993a5f-ed28-49ad-a0db-ace4cc2312f9)

gabrielfvale commented:
Looks great. I also dig the template with description + assets used, so thanks!
